vendor.conf

```diff
@@ -63,9 +63,9 @@ github.com/coreos/pkg v3
 github.com/golang/groupcache b710c8433bd175204919eb38776e944233235d03
 github.com/fsnotify/fsnotify 7d7316ed6e1ed2de075aab8dfc76de5d158d66e1
 github.com/emicklei/go-restful 09691a3b6378b740595c1002f40c34dd5f218a22
-github.com/Azure/go-ansiterm fa152c58bc15761d0200cb75fe958b89a9d4888e
-github.com/Microsoft/go-winio v0.3.7
-github.com/Microsoft/hcsshim v0.5.11
+github.com/Azure/go-ansiterm master
```

Comment:
We should use specific commits instead of master so it is easier to debug if there are issues.